tests/unit/autocomplete/helper.js

```diff
@@ -0,0 +1,26 @@
+define( [
+	"intern!qunit",
+	"jquery",
+	"lib/helper",
+	"lib/css",
+	"text!tests/unit/autocomplete/autocomplete.html",
+	"ui/accordion",
+	"jquery-simulate",
+	"./common"
+], function( QUnit, $, helper, cssjs, htmlContent ) {
+cssjs( { module: "core menu autocomplete" } );
+
+return $.extend( helper, {
+	setupTeardown: function() {
+		return {
+			setup: function() {
+				$("body").append(htmlContent);
+			},
+			teardown: function() {
+				$("#qunit-fixture").remove();
+			}
+		};
+	}
+} );
+
+} );
```

Comment:
This entire file is what I want to get rid of. We've put a lot of work into not having this duplicated over and over and over.